llvm/unittests/SandboxIR/SandboxIRTest.cpp

```diff
@@ -2891,7 +2891,8 @@ define void @foo(i1 %cond0, i1 %cond2) {
 
   {
     // Check unconditional BranchInst::create() InsertBefore.
-    auto *Br = sandboxir::BranchInst::create(BB1, /*InsertBefore=*/Ret1, Ctx);
+    auto *Br = sandboxir::BranchInst::create(
+        BB1, /*InsertBefore=*/Ret1->getIterator(), Ctx);
```

Comment:
The `/*InsertBefore=*/` comment should now be `/*InsertPosition=*/`, right? Same for other occurrences in this PR.

Reply:
Ah yeah, there are a couple of them left, I forgot to fix them, thanks for noticing it!